distribution/docs/src/main/resources/_contents/_security-contents/managing-security-contents.adoc

```diff
@@ -83,12 +83,52 @@ From the Web Admin Console, the following configurations are available from a st
 |`org.codice.${ddf-branding-lowercase}security.filter.login.Session`
 |Set session timeout
 
-|Web Context Policy Manager
+|<<web_context_policy_manager_table, Web Context Policy Manager>>
 |`org.codice.${ddf-branding-lowercase}security.policy.context.impl.PolicyManager`
 |Configure Realms, Auth types, and Attributes for different contexts.
 
 |===
 
+.[[web_context_policy_manager_table]]Web Context Policy Manager Configuration
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+
+|Context Realms
+|realms
+|String
+|List of realms supporting each context. Provided realms are: DDF, karaf. Example: /=DDF
```

Comment:
This is wrong, the only realm provided out of the box is karaf. Installing our ldap login module will also add an ldap realm